pandas/core/groupby/groupby.py

```diff
@@ -1294,12 +1294,118 @@ def describe(self, **kwargs):
                 return result.T
             return result.unstack()
 
-    @Substitution(name='groupby')
-    @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
         """
-        Provide resampling when using a TimeGrouper
-        Return a new grouper with our resampler appended
+        Provide resampling when using a TimeGrouper.
+
+        Given a grouper, the function resamples it according to a string
+        "string" -> "frequency".
+
+        See the :ref:`frequency aliases <timeseries.offset-aliases>`
+        documentation for more details.
+
+        Parameters
+        ----------
+        rule : str or Offset
+            The offset string or object representing target grouper conversion.
+        *args, **kwargs
+            For compatibility with other groupby methods. Available keywor
+            arguments are:
```

Comment:
typo keywor --> keyword

Reply:
Sorry, these lines doesn't seem to be there anymore in the code.